src/views/empty-state.html

```diff
@@ -2,13 +2,13 @@
   <div ng-if="$ctrl.config.icon" class="blank-slate-pf-icon">
     <span class="{{$ctrl.config.icon}}"></span>
   </div>
-  <h1 id="title">
+  <h4 id="empty-state-title-{{$id}}" class="h1 title">
```

Comment:
`h1` is just [the bootstrap way](https://getbootstrap.com/docs/3.3/css/#type-headings) to style a different heading as an `h1`, thats the only reason i kept it.  I don't know that we need to keep the class `.title` either, i just kept it so that the test updates are changes from `#title` to `.title`.  Def open to any suggestions, was just going for the most minimal changes to make this work!